src/main/java/com/github/princesslana/eriscasper/util/QueryStringBuilder.java

```diff
@@ -26,6 +26,10 @@ public QueryStringBuilder addSnowflake(String name, Optional<Snowflake> snowflak
     return snowflake.map(s -> add(name, s.unwrap())).orElse(this);
   }
 
+  public QueryStringBuilder addBoolean(String name, Optional<Boolean> value) {
+    return value.map(v -> add(name, v.toString())).orElse(this);
```

Comment:
Are we sure this outputs what we need? i.e., `true` vs `True` or something else?

Reply:
https://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html I'm sure, needs true/false